### PR TITLE
[libc] Include missing RISC-V stdlib.h and math.h entrypoints

### PR DESCRIPTION
**Pull request by petrhosek**
This matches the entrypoints for baremetal ARM.

### CODE REVIEW

**Review by nickdesaulniers — COMMENTED** [review]
Please also update the tables at libc/docs/math/index.rst

have you verified that the unit tests pass on these targets?

**Comment by petrhosek** [comment]
> Please also update the tables at libc/docs/math/index.rst

Done.

> have you verified that the unit tests pass on these targets?

No, we currently have a bit of a chicken-and-egg problem where we need these targets to build the SDK which we need to run unit tests on the target.

**Comment by nickdesaulniers** [comment]
> No, we currently have a bit of a chicken-and-egg problem where we need these targets to build the SDK which we need to run unit tests on the target.

no worries, we also don't have a riscv32 buildbot set up yet, so we will be reliant on the results of your testing for now. We appreciate it (and need to consider more how to remedy the lack of buildbots for that target). Please send any fixups necessary (or file bugs).

File: libc/docs/math/index.rst [hunk 1]
```diff
@@ -108,171 +108,171 @@ Basic Operations
...
+| trunc        | |check| | |check| | |check| | |check| | |check| |         |         | |check| | |check| | |check| |         |         |
 +--------------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+
-| truncf       | |check| | |check| | |check| | |check| | |check| |         |         | |check| | |check| |         |         |         |
+| truncf       | |check| | |check| | |check| | |check| | |check| |         |         | |check| | |check| | |check| |         |         |
 +--------------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+
-| truncl       | |check| | |check| | |check| | |check| | |check| |         |         | |check| | |check| |         |         |         |
+| truncl       | |check| | |check| | |check| | |check| | |check| |         |         | |check| | |check| | |check| |         |         |
 +--------------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+
```
Comment (nickdesaulniers) [inline libc/docs/math/index.rst:278]: I can't comment below, but please update the second table in this file, too (below)!

Comment (petrhosek) [reply libc/docs/math/index.rst:278]: Done, sorry about that I missed it earlier.

**Review by lntue — APPROVED** [review]

**Comment by mikhailramalho** [comment]
> > No, we currently have a bit of a chicken-and-egg problem where we need these targets to build the SDK which we need to run unit tests on the target.
> 
> no worries, we also don't have a riscv32 buildbot set up yet, so we will be reliant on the results of your testing for now. We appreciate it (and need to consider more how to remedy the lack of buildbots for that target). Please send any fixups necessary (or file bugs).

We do have a rv32 buildbot on staging: https://lab.llvm.org/staging/#/builders/149 but it seems to be broken since the werror patch

I think the current error is that `-fuse-ld=lld` is being passed to `CMAKE_CXX_FLAGS` to `CMAKE_C_FLAGS`, instead of `LD_FLAGS`. After that there are some error of shortening 64-to-32

**Comment by nickdesaulniers** [comment]
> We do have a rv32 buildbot on staging

Oh, interesting!  What's staging and why do we have that there? https://lab.llvm.org/buildbot/#/builders?tags=libc is the dashboard I'm usually looking at, and that has an rv32 build bot that seems offline.

**Pull request merged**